CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.4.13
+
+* Use VERSION file when generating docs.
+* 64-bit gcc on Windows build error
+* New: setZerosOnOuter, increaseRowNonZeroCountsOnOuterTo
+* Longer docstrings on new methods
+* Remove a nupic.math test that should stay in nupic
+* Use unittest, not unittest2, to match the nupic.core CI config
+* Add C++ unit tests for the new SparseMatrix methods
+* Finish moving nupic.bindings math tests to nupic.core
+* s/AboveAndBelow/BelowAndAbove
+* Unit tests for new SparseMatrix methods
+* Fix changes due to RNG change in nupic.core.
+* Changed all copyright headers on all files to AGPL.
+* Removes StdRandom class that was defined in math.i but never used.
+* Tests folder refactored
+* Update unit test for swigged shuffle function.
+* Remove assertSequenceEqual that doesn't work with 2.6
+* Python 2.6 fix
```

Comment:
I looked into this. First shock was that I supposedly authored the change a few weeks ago. But then I found that it was part of a set of commits that were pulled in when Marcus moved tests from `nupic` to `nupic.core` and included the history with them. So in reality, the 2.6 changes were made a long time ago when we still targeted that version (for Jython compatibility). I'll remove the notes form the changelog.